fe/fe-core/src/main/java/com/starrocks/persist/Storage.java

```diff
@@ -117,30 +104,29 @@ public void reload() throws IOException {
         }
 
         // Find the latest image
-        File dir = new File(metaDir);
+        imageSeq = getImageSeq(metaDir);
+    }
+
+    public static long getImageSeq(String imageDir) {
```

Comment:
You should unite journal id and image seq to one name to keep our concept consistent.

Reply:
Change  imageSeq to ImageJournalId?